lib/graph/treeshake.js

```diff
@@ -17,6 +18,9 @@ function treeshake(data) {
 	let getNode = id => data.graph[id];
 	let mains = uniq(data.mains.concat(data.loader.bundle));
 	let cwd = process.cwd().substr(1);
+	if (os.type().toLowerCase() === "windows_nt"){
```

Comment:
Is this the best way to check if it's windows? In the past I've used `process.platform === 'win32'`.

Reply:
hmm i dont know. docs said `Windows_NT` https://nodejs.org/api/os.html#os_os_type

stackoverflow said  `process.platform` https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js

Reply:
Ok, I'd think either will work then.